packages/x-date-pickers-pro/src/index.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-restricted-imports
```

Comment:
This was the only case we had this line for: https://github.com/mui/mui-x/pull/18643/files#diff-9601a8f6c734c2001be34a2361f76946d19a39a709b5e8c624a2a5a0aade05f2L92